R/plotting.R

```diff
@@ -908,3 +908,55 @@ getMarkerCols <- function(object, fcol = "markers") {
     col[ukn] <- getUnknowncol()
     col
 }
+
+
+##' The function plots marker consensus profiles obtained from mrkConsProfile
+##'
+##' @title Plot marker consenses profiles.
+##' @param object A \code{matrix} containing marker consensus profiles as output
+##'   from \code{\link{mrkConsProfiles}}.
+##' @param order Order for markers (Optional).
+##' @param plot A \code{logical} defining whether the heatmap should be plotted.
+##'   Default is \code{TRUE}.
+##' @return Invisibly returns \code{\link[ggplot2]{ggplot}} object.
+##' @author Tom Smith
+##' @examples
+##' library("pRolocdata")
+##' data(E14TG2aS1)
+##' hc <- mrkHClust(E14TG2aS1, plot=FALSE)
+##' mm <- getMarkerClasses(E14TG2aS1)
+##' m_order <- levels(factor(mm))[order.dendrogram(hc)]
+##' fmat <- mrkConsProfiles(E14TG2aS1)
+##' plotConsProfiles(fmat, order=m_order)
+plotConsProfiles <- function(object, order=NULL, plot=TRUE){
+
```

Comment:
To fix the note in the check, could you create the dummy variable `feature <- NULL`. 

For your information, there's no such note for `intensity` and `sample` because these symbols already exist.